### PR TITLE
Add the basic structure of the leadboard.

### PR DESCRIPTION
**Pull request by saikatgpae**
This PR is created to create the basic structure of the leadbord exercise.

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -7,6 +7,28 @@
...
+        </section>
+        <section class="add-score">
+            <h3>Add your score</h3>
+            <form action="#" class="d-flex flex-column">
+                <input type="text" placeholder="Your name" class="m-3">
+                <input type="text" placeholder="Your score" class="m-3">
+                <button class="m-3">Submit</button>
+            </form>
+        </section>
+    </main>
```
Comment (ShahierNashaat) [inline src/index.html]: - [x] Please make your design like the wireframe as you should make the scores table wider and the submit button to the right.

Your Design | Wireframe Design
------------- | --------------------
Capture3 | leaderboard_wireframe

**Review by ShahierNashaat — CHANGES_REQUESTED** [review]
Hi @saikatgpae, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 💯 

- Correct use of GitFlow ✔️
- Descriptive commit messages ✔️
- Professional looking pull request ✔️
- Your readme looks professional ✔️
- Your code is clean and well documented ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.html (hunk 1 above)
Comment (saikatgpae) [reply src/index.html]: Thank you for your suggestion.
Changed the same.

**Review by muneebulrehman — APPROVED** [review]
Hi @saikatgpae , 

Your project is complete!  There is nothing else to say other than... it's time to merge it 🥳
Congratulations! 🎉 

# Status: Approved✔️

## Highlights
- No linter errors
- Good follow-up with the previous reviewer.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**